src/sentry/tasks/post_process.py

```diff
@@ -595,6 +595,37 @@ def process_rules(job: PostProcessJob) -> None:
     return
 
 
+def process_code_mappings(job: PostProcessJob) -> None:
+    if job["is_reprocessed"]:
+        return
+
+    from sentry.tasks.derive_code_mappings import derive_code_mappings
+
+    try:
+        event = job["event"]
+        project = event.project
+
+        cache_key = f"code-mappings:{project.id}"
+        project_queued = cache.get(cache_key)
+        if project_queued is None:
+            cache.set(cache_key, True, 3600)
+
+        if project_queued or not features.has(
+            "organizations:derive-code-mappings", event.project.organization
+        ):
+            return
+
+        if killswitch_matches_context(
```

Comment:
Is this the right way to use a killswitch?

Reply:
I don't know. Have a look at the blame page and see who can confirm.